lib/rpc.ts

```diff
@@ -83,10 +83,13 @@ interface IForwardingName {
   name: string;
 }
 
+export type ICallWrapper = (call: () => any) => any;
```

Comment:
I suggest renaming `call` to `callFunc`. I'd say also this is a good place to define an identity wrapper like `function plainCall(callFunc: () => any) { return callFunc(); }`

Also, maybe it's better to replace `any` with `Promise<any>`? Otherwise this wrapper is essentially disabling all type-checking of return values.